Add KUBECONFIG check to local setup script to avoid using kcp admin config incorrectly

<!--- Please identify the urgency in addressing this issue --->
Priority: Low

## Issue Description
If you run `make local-setup` in a terminal where you have exported the KUBECONFIG to point to the kcp admin kubeconfig (`export KUBECONFIG=.kcp/admin.kubeconfig`), it will fail with a confusing error:

```
error: current URL "https://127.0.0.1:46195" does not point to cluster workspace
Killing KCP
make: *** [Makefile:174: local-setup] Error 1
```

The local-setup script expects that you are adding your physical clusters (kind) using either your default KUBECONFIG `~/.kube/config`, or any config other than `.kcp/admin.kubeconfig`

## Suggested Resolution

Add a check to local-setup to make sure KUBECONFIG is not set to `.kcp/admin.kubeconfig`

--- a/utils/local-setup-add-crc-cluster.sh
+++ b/utils/local-setup-add-crc-cluster.sh
@@ -55,6 +55,12 @@ KUBECTL_KCP_BIN="./bin/kubectl-kcp"
 : ${KCP_VERSION:="release-0.8"}
 KCP_SYNCER_IMAGE="ghcr.io/kcp-dev/kcp/syncer:${KCP_VERSION}"
 
+#Check that KUBECONFIG is not set to .kcp/admin.kubeconfig
+if [[ $KUBECONFIG == "${KUBECONFIG_KCP_ADMIN}" ]] ; then
+  echo "Can NOT add physical clusters (kind) if KUBECONFIG is set to .kcp/admin.kubeconfig"
+  exit 1 #this will trigger cleanup function
+fi
+
 crc start -p $PULL_SECRET
 
 cp ~/.crc/machines/crc/kubeconfig ${TEMP_DIR}/${CRC_KUBECONFIG}

--- a/utils/local-setup.sh
+++ b/utils/local-setup.sh
@@ -155,6 +155,12 @@ if ! [[ $clusterCount =~ "0" ]] ; then
   ${KIND_BIN} get clusters | grep ${KIND_CLUSTER_PREFIX} | xargs ${KIND_BIN} delete clusters
 fi
 
+#Check that KUBECONFIG is not set to .kcp/admin.kubeconfig
+if [[ $KUBECONFIG == "${KUBECONFIG_KCP_ADMIN}" ]] ; then
+  echo "Can NOT add physical clusters (kind) if KUBECONFIG is set to .kcp/admin.kubeconfig"
+  exit 1 #this will trigger cleanup function
+fi
+
 #1. Start KCP
 echo "Starting KCP, sending logs to ${KCP_LOG_FILE}"
 ${KCP_BIN} --v=9 start --run-controllers > ${KCP_LOG_FILE} 2>&1 &

